arroyo-controller/src/lib.rs

```diff
@@ -447,6 +448,18 @@ impl ControllerServer {
             .create_pool(Some(deadpool_postgres::Runtime::Tokio1), NoTls)
             .unwrap();
 
+        // test that the DB connection is valid
+        let _ = pool.get().await.unwrap_or_else(|e| {
+            panic!(
```

Comment:
Is it standard to panic in these cases, rather than either returning an error or exiting?

Reply:
Panicing will produce an error and exit. Eventually we'll probably want clearer error handling for users, but while most of our users are going to be developers working on the system I think it's more helpful to have the context that a panic gives you (like the line).